.travis.yml

```diff
@@ -3,35 +3,25 @@
 language: node_js
 matrix:
   include:
+    # Test on latest version of node
+    - node_js: 'node'
 
-    # Run lint only in Node.js 0.12
-    - node_js: '0.12'
+    # Run lint only in the latest LTS
+    - node_js: '6'
```

Comment:
Right, I realise this probably should be better documented.

* `node_js: 'node'` runs the tests on the latest stable version of node available on Travis (`7.7.1` at the time of writing). This _fixes_ the issue that we discussed last week regarding what to do when Node 8 is released - this will run the tests on whatever version is the latest so it will upgrade from 7 to 8 without any manual intervention and it doesn't force us to add node 7 to the test matrix, which is a solution that I heavily disagree with.
* The linter will run on node 6 (which fixes the bug of xo failing to run on node 0.12, and gives us some leeway for when node 4 is deprecated)
* Rest of the tests as they were already specified - I've just changed the order.